scss/_variables.scss

```diff
@@ -1001,8 +1001,8 @@ $input-placeholder-color:               $gray-700 !default;
 $input-plaintext-color:                 var(--#{$prefix}body-color) !default;
 
 $input-height-inner:                    add($input-line-height * 1em, $input-padding-y * 2) !default;
-// Boosted mod: no $input-height-inner-half
-// Boosted mod: no $input-height-inner-quarter
+$input-height-inner-half:               $spacer !default; // Boosted mod
```

Comment:
Why don't we use the same values as Bootstrap?
```scss
$input-height-inner-half:               add($input-line-height * .5em, $input-padding-y) !default;
$input-height-inner-quarter:            add($input-line-height * .25em, $input-padding-y * .5) !default;
```

Reply:
Let's keep it as you developed it for now. We'll tackle this issue for precisely when the design for form validation will be in the DSM and final.